google-cloud-tools-plugin/testData/descriptor/appengine-web_runtime-java8-serviceandmodule.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2017 Google Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+    <application>app</application>
+    <version>1</version>
+    <runtime>java8</runtime>
+    <service>java8Service</service>
```

Comment:
Oh, hah, that's because I realised I hadn't added unit tests for the getServiceNameFromAppEngineWebXml() method.

I guess it won't matter after I merge this with the flexible-facet branch.